app/classifier/tasks/survey/choice.cjsx

```diff
@@ -44,15 +44,23 @@ ImageFlipper = React.createClass
 module.exports = React.createClass
   displayName: 'Choice'
 
+  propTypes:
+    annotationValue: React.PropTypes.shape({
+      answers: React.PropTypes.object,
+      choice: React.PropTypes.string,
+      filters: React.PropTypes.object
+    })
+
   getDefaultProps: ->
+    annotationValue: { answers: {} }
     task: null
     choiceID: ''
     onSwitch: Function.prototype
     onConfirm: Function.prototype
     onCancel: Function.prototype
 
   getInitialState: ->
-    answers: {}
+    answers: @props.annotationValue.answers
```

Comment:
If this component mounts before the annotation has updated in the parent, then the value here will be from the previous annotation.

I think the solution to that is to check `annotationValue` in `componentWillReceiveProps` and update state if necessary.